src/main/java/org/jabref/logic/importer/fetcher/ArXiv.java

```diff
@@ -367,8 +367,8 @@ public static String correctLineBreaks(String s) {
          * Returns the arXiv identifier
          */
         public Optional<String> getIdString() {
-            // remove leading https://arxiv.org/abs/ from abstract url to get arXiv ID
-            String prefix = "https://arxiv.org/abs/";
+            // remove leading http://arxiv.org/abs/ from abstract url to get arXiv ID
+            String prefix = "http://arxiv.org/abs/";
```

Comment:
Why no HTTPS anymore? We just changed this?!

Reply:
I am also in favor of removing both http and https prefixes

Reply:
The fetcher itself still uses https, but the prefix for the eprint field does not start with https but with http

Reply:
But we only remove http prefixes not https (if there exists one at a point in time)?!

Reply:
Added check for both